micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -0,0 +1,222 @@
+# Changelog 
+
+## [1.8.13]  [16-Dec-2024]
+- Made validations for name, mobile number, and password fields in user profile update screen configurable through MDMS data
+  - Implemented dynamic regex validation for profile updates based on MDMS data
+  - Added support for custom regex patterns through `UserProfileValidationConfig`
+  - Example MDMS data: https://github.com/egovernments/egov-mdms-data/blob/UNIFIED-QA/data/mz/commonUIConfig/UserProfileValidationConfig.json
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)